src/FirstSteps/RPG/Heroes/HeroesCreator.cs

```diff
@@ -28,5 +30,29 @@ public static Hero Create(string name, Races race)
                 return null;
             }
         }
+        public static Hero Create(Races race)
```

Comment:
As I wrote above, `Hero LoadFromFile()` could be a better method for loading a hero.